.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Please remove these intellij files.
I think they should have already been ignored in .gitignore